packages/idx/src/idx.ts

```diff
@@ -1446,6 +1449,10 @@ const angularJsService = (
         }
     ): Model<T> {
         // request.direct = true;
+        if (request.api && !isString(request.api)) {
+            // Ensure we are requesting the apiVersion these widgets were designed for
+            request.api.apiVersion = apiVersion
```

Comment:
I would also add conditionals for `typeof request.api === 'object' &&  !Array.isArray(request.api)` to this `if` statement and the one below, unless you're certain of the typing here.

Reply:
actually need to Type control those and I've yet to